scripts/generator/graphql_generator/csharp/Cart.cs.erb

```diff
@@ -102,6 +103,52 @@ namespace <%= namespace %> {
             });
         }
 
+        public void SetShippingLine(string shippingRateHandle, Action<ShopifyError> callback) {
```

Comment:
Would the SDK user ever use these methods or are they made public for `INativeCheckout` to use them?